### PR TITLE
fix(translation): Update color blind mode location in FAQ answer for EN and ID language 

### LINKED ISSUE #6780: Location of the color blind mode switch, as stated in FAQ, seems inaccurate
The [FAQ](//app.electricitymaps.com/faq) says it's at the bottom of the left panel. I couldn't find it there, but there is a toggle among the icons at the right of the screen.

The FAQ says: I am color blind. How can I read your map?
You can change the colours displayed in the map by enabling the “color blind mode” setting. It is located on the bottom of the left panel on desktop and in the information tab on the mobile app., but the icon is actually in the row of icons at the right. (Image indicates this with yellow marker, red arrow and red circle.)

### PR DESCRIPTION
**Pull request by qalbun-salim**
## Issue
Closes #6780 
Note: We need to update translations for other language as well, so we may need to create issue for other languages or not close this one yet 🙇 

## Description
This PR changes  the answer in FAQ section for color blind question. The location of color blind mode is updated to map control in desktop and settings tab in mobile

### Preview
EN language: 
image

ID language:
image

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/src/locales/en.json [hunk 1]
```diff
@@ -374,7 +374,7 @@
...
     "mapNuclearColors-answer": "Nuclear energy production has a very low carbon intensity (<a href=\"https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources\" target=\"_blank\" >source</a>), and nuclear-powered countries are therefore shown as green on the map. This does not mean that there are not other environmental challenges associated with nuclear energy (as is also the case with other forms of energy production), but these challenges are unrelated to the emissions of greenhouse gases and are therefore outside the scope of this map.",
     "mapColorBlind-question": "I am color blind. How can I read your map?",
-    "mapColorBlind-answer": "You can change the colours displayed in the map by enabling the “color blind mode” setting. It is located on the bottom of the left panel on desktop and in the information tab on the mobile app."
+    "mapColorBlind-answer": "You can change the colours displayed in the map by enabling the “color blind mode” setting. It is located below the select language button in the map control on desktop and in the settings tab on the mobile app."
```
Comment (silkeholmebonnen) [inline web/src/locales/en.json:377]: I'm not sure that the average user know where to look for the map control. Maybe we could say that it is located to the right, below the select language button?

Comment (qalbun-salim) [reply web/src/locales/en.json:377]: Noted, sorry for the late response 🙇  I agree that map control may not sound intuitive enough on first sight. will update it in new commit 👍

Comment (phiphou) [reply web/src/locales/en.json]: Let me know for it so I would close my PR.

Comment (phiphou) [reply web/src/locales/en.json]: @qalbun-salim Let me know for it so I would close my PR.

**Review by silkeholmebonnen — APPROVED** [review]
Nice work! 🌟

**Pull request merged**